### PR TITLE
Make the changes from doc fix persistant

### PR DESCRIPTION
**Pull request by nickvergessen**
https://github.com/nextcloud/documentation/commit/6ccd2c5678b85d35a8673d784bef5c6c89e7ecb9 

Needs backport + script run

@MorrisJobke @LukasReschke

### CODE REVIEW

**Comment by icewind1991** [comment]
:+1:

**Comment by MorrisJobke** [comment]
👍

File: config/config.sample.php [hunk 1]
```diff
@@ -435,7 +435,7 @@
  * Both minimum and maximum times can be set together to explicitly define
  * file and folder deletion. For migration purposes, this setting is installed
  * initially set to "auto", which is equivalent to the default setting in
- * ownCloud 8.1 and before.
+ * Nextloud.
```
Comment (LukasReschke) [inline config/config.sample.php]: Nextloud? ;)

Comment (nickvergessen) [inline config/config.sample.php]: Fixed in #517

**Pull request merged**